packages/zosjobs/src/GetJobs.ts

```diff
@@ -180,12 +181,6 @@ export class GetJobs {
                 }
                 query += JobsConstants.QUERY_JOBID + encodeURIComponent(params.jobid);
             }
-            if (params.execData) {
-                if (RestClient.hasQueryString(query)) {
-                    query += JobsConstants.COMBO_ID;
-                }
-                query += JobsConstants.EXEC_DATA;
-            }
```

Comment:
Can we add this block back and remove the `if` block from lines 192-197? I think we'll want to keep the default behavior of only fetching exec data when the parameter is passed.